<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionInfoDlg</class>
 <widget class="QDialog" name="VersionInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Version Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Secure Processor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maxim Integrated MAX32550</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>83</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>32550 Boot:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>32550 Os:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>153</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel version:   4.1.15</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>87</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Uboot:  1.0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>122</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Master App:  1.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBoot">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelOs">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>94</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>i.MX 6UltraLit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>152</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Application Processor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Firmware Version:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
